po/pt.po

```diff
@@ -1,131 +1,160 @@
-# Portuguese translations for com.github.cassidyjames.ideogram package.
-# Copyright (C) 2019 THE com.github.cassidyjames.ideogram'S COPYRIGHT HOLDER
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
```

Comment:
Why change these lines?

Reply:
I think it's because i used the .pot file to make the translation and not the pt.po file.